protocol/contracts/amo/Ramos.sol

```diff
@@ -343,9 +347,6 @@ contract RAMOS is Ownable, Pausable {
         unchecked {
             bptIn = bptAmountAfter - bptAmountBefore;
         }
-        if (bptIn < minBptOut) {
-            revert AMOCommon.InsufficientAmountOutPostcall(minBptOut, bptIn);
-        }
 
         // stake BPT
         bptToken.safeTransfer(address(amoStaking), bptIn);
```

Comment:
We might want to take the opportunity to add some new events here. eg
  `emit LiquidityAdded(stableAdded, templeAdded, bptReceived)`
Another team member (Dariox) is a subgraph king and struggled a bit with a few missing events.